youtube_dl/extractor/pbs.py

```
@@ -465,8 +467,13 @@ def _extract_webpage(self, url):
             player_page = self._download_webpage(
                 url, display_id, note='Downloading player page',
                 errnote='Could not download player page')
-            video_id = self._search_regex(
-                r'<div\s+id="video_([0-9]+)"', player_page, 'video ID')
+            
+            try:
+                video_id = self._search_regex(
+                    r'<div\s+id="video_([0-9]+)"', player_page, 'video ID')
+            except:
```

Never use bare except. Use list of regexes in `_search_regex` instead.